charts/collabora-online/0.0.1/questions.yaml

```
@@ -0,0 +1,332 @@
+                              
+  # environmentVariables Configuraiton
+  - variable: env
+    group: "Environment Variables"
+    label: "Environment Variables Configuration"
+    schema:
+      type: dict
+      required: true
+      attrs:
+        - variable: domain
+          label: "Domains will be using collabora"
+          description: 'Use backslash "\" before dots ".". Use pipe "|" to separate multiple domains'
+          schema:
+            type: string
+            default: 'nextcloud\.domain\.tld|othernextcloud\.domain\.tld'
+            required: true
+        - variable: username 
+          label: "Username for WebUI"
+          schema:
+            type: string
+            default: "admin"
+            required: true
+        - variable: password
+          label: "Password for WebUI"
+          schema:
+            type: string
+            private: true
+            default: "changeme"
+            required: true
+        - variable: dictionaries
+          label: "Dictionaries to use, leave empty to use all"
+          schema:
+            type: string
+            default: "de_DE en_GB en_US es_ES fr_FR it nl pt_BR pt_PT ru"
+        - variable: extra_params
+          label: "Extra Parameters to add"
+          description: 'e.g. "–o:welcome.enable=false", See more on /etc/loolwsd/loowsd.xml. Separate params with space'
+          schema:
+            type: string
+            default: "-o:welcome.enable=false -o:user_interface.mode=notebookbar"        
+
+        # Adding those makes install fail. Maybe because of the empty value?---------------
+
+        # Maybe we must put those (cert_domain, server_name, DONT_GEN_SSL_CERT) under a Tick-box, like "Enable Advanced Options"
+        # - variable: cert_domain
+        #   label: "Domain to generate SSL (Basic setup probably won't need this)"
+        #   description: "When this environment variable is set (is not “”), then startup script will generate a new SSL certificate signed by a dummy CA for this domain, not for localhost"
+        #   schema:
+        #     type: string
+        #     default: ""    
+        # - variable: server_name
+        #   label: "Server Name (Basic setup probably won't need this)"
+        #   description: "When this environment variable is set (is not “”), then its value will be used as server name in /etc/loolwsd/loolwsd.xml. Without this, CODE is not delivering a correct host for the websocket connection in case of a proxy in front of it."    
+        #   schema:
+        #     type: string
+        #     default: ""
+        # - variable: DONT_GEN_SSL_CERT
+        #   label: "Don't generate SSL cert"
+        #   description: "When this environment variable is set (is not “”), then startup script will not generate a new SSL certificate signed by a dummy CA. It is useful, if you want to use your own SSL certificate for some reason."   
+        #   schema:
+        #     type: string
+        #     default: ""
```

We do not support internal/nodePort SSL (well, not actually valid SSL).
Use traefik for it instead.

```suggestion
        # - variable: server_name
        #   label: "Server Name (Basic setup probably won't need this)"
        #   description: "When this environment variable is set (is not “”), then its value will be used as server name in /etc/loolwsd/loolwsd.xml. Without this, CODE is not delivering a correct host for the websocket connection in case of a proxy in front of it."    
        #   schema:
        #     type: string
        #     default: ""
```